### DISCUSSION TITLE
Is there notifications which shows up in metamask popup itself

### DISCUSSION THREAD

**Discussion by Arch0125** [MetaMask/snaps#1238] [Q&A] [answered]
I kind of remember coming across something called metamask_notifs, i'm not sure if it exists, if it does it would be great if you guys drop a link to it

**Comment by Montoya** [comment, marked as answer]
You can use the notifications API in Snaps to send notifications that appear inside the wallet: https://docs.metamask.io/guide/snaps-rpc-api.html#snap-notify

**Reply by Arch0125** [reply]
I guess its not possible to send these to another wallet address from mine ?
Adding to this, I cannot actually import and use any external npm packages inside snaps.ts, is there any workaround as of now ?

**Reply by Arch0125** [reply]
Also could you drop a snippet to give permissions to snap to call restricted methods as im not being able to figure it out from docs

**Reply by Montoya** [reply]
Here is an example snap.manifest.json for a notifications snap: 
https://github.com/MetaMask/test-snaps/blob/main/packages/notification/snap.manifest.json

Note that the `initialPermissions` object includes `"snap_notify": {}`. 

Also, to send notifications to another wallet you would need a cloud service to route messages between wallets. So you would also need `"endowment:network-access": {}` to send messages to a server and check the server for messages for an address.
